Gruntfile.js

```diff
@@ -44,7 +49,7 @@ module.exports = function (grunt) {
         },
         concurrent: {
             watch: {
-                tasks: ["watch:serve", "watch:client"],
+                tasks: ["watch:serve", "webpack:watch"],
```

Comment:
I think it'd probably be better to keep this running watch:client, and add "build" to the watch:client task list, to make sure that the linter (and anything else we might want to add to the process later) is included, and also so that the targeted watch event below is fired properly to reduce linting times